doc/dependencies.md

```diff
@@ -25,6 +25,7 @@ These are the dependencies currently used by Bitcoin Core. You can find instruct
 | xkbcommon |  |  |  |  | [Yes](https://github.com/bitcoin/bitcoin/blob/master/depends/packages/qt.mk) (Linux only) |
 | ZeroMQ | [4.3.1](https://github.com/zeromq/libzmq/releases) | 4.0.0 | No |  |  |
 | zlib | [1.2.11](https://zlib.net/) |  |  |  | No |
+| SQLite 3 | [3.32.1](https://sqlite.org/download.html) |  |  |  |  |
```

Comment:
05c427a0654d2fe0907d8a755e74a2e2d0a51f07
1. Mind keeping the lexicographic order?

2. Why minimum version is not mentioned?

Reply:
The minimum version is not known.